### DISCUSSION TITLE
I recieved zero warnings of the impending removal of Classic User Projects

### DISCUSSION THREAD

**Discussion by SandbergSan** [community/community#137607] [Projects and Issues] [answered]
### Select Topic Area

General

### Feature Area

Projects

### Body

I just experienced all my projects being removed from Classic and moved to the New Project system, and had zero warnings about this happening. I get a warning now, AFTER it happened, which makes no sense. Can you please be more careful about doing stuff like this... Now i manually have to go and fix all the errors the migration has caused (for example removing todo stuff that is over one year old) and i have lost all my faith in github, who i have been a fan of using for years. Seriously whoever took this decision should be fired. The new Project feature SUCKS, that is why so many are still were using the Classic one. Make the new one better than the old one before you force people over to it. I am so disappointed and i will never recommend people to use github after a blunder this big.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by evi-liu** [comment]
Hi there, could you share the link to the project that was migrated? If you'd like, we can manually migrate the project to ensure all of your cards are migrated.

In addition, could you tell us more about why you're dissatisfied with the new Projects experience? Your feedback is valuable to us as we work to improve our product.

**Reply by SandbergSan** [reply]
I have spent the last 24 hours re-assigning the issues of the now removed Classic Project as new cards in the new Project, so i rather not say what project it is so you go and re-migrate it and mess everything up. It is all sorted now, and it is quite annoying to sort out 700 issues by hand, so i dont want to risk having to do it again.

The main issues with the sunsetting for me was:
1: The migration would have been fine if you just migrated ALL cards on the kanban board. Why did you put a time limit of one year for them to be discarded? I have projects on the kanban boards that has been running since 2020 and they are still active today, and there were still cards of "To Do"s and stuff that i havent gotten time for yet, or Ideas that could be implemented in future.
2: I had no idea the sunsetting was happening. I was not informed on the page in any way that it was going to get removed. I dont check these forums or twitter or anything, but i dont think that should be required to get to know about this a fundamental change like this happening. The only note i got on the page was yesterday, after the migration already had happened, and my memory is abit fuzzy, but it was like this: "Hi... we just migrated those projects you had and removed 80% of all cards from your kanban board... Have a nice day!"


The main issue with the New Projects are, albeit small, but annoying as a Classic Project user:
In the Classic Project setup, i could sit with the kanban board and do everything from there. I converted the notes to an issue and it converted automaticly to the repository that the project was linked to. In the new one, when converting the "draft" note to an issue, i have to pick the repository every single time, and it is so annoying when you are used to the Classic Projects.

And yes... You are going to say..."You can automate this with the new "Workflows" in the projects, but there are two issues with that:

1. That automation requires issues to be created from the "Issues" page, and that would mean that i would have to go back and forth between those two pages, and I would prefer to just use the kanban board, and create issues from there.
2. I CANT use the Issues page because i have multiple projects in one repository. So if i create an issue on the Issues page, it would either mean i would have to tag them differently depending on what project i want it to automaticly be moved to, or just have all Issues automaticly end up in one specific project, and then move them later if it was supposed to end up in the other projects. So that solution is even more bunch of clicks just to get the Issue on the correct kanban board.

Either way, it is more work than just converting the note to an issue like in the Classic Project, where it was one click.

My suggestion for a solution you could implement is this:

In the settings for the project have a setting called: Auto-Assign Repository (default turned off, of course) 
If that setting is turned on, there is a dropdown next to it that gets enabled. That dropdown is the very same dropdown that shows up whenever you convert a "draft" card to an issue on the kanban board. The difference is that now you are picking it for the project. After this setting being enabled, the project will automaticly pick that whenever i convert a "draft" card to an issue, so that i dont have to do it every single time i convert a draft to an issue.

**Reply by evi-liu** [reply, marked as answer]
This is excellent feedback, thank you! I definitely hear you that it would be nice to convert drafts to issues with one click. We have some improvements coming soon for the creating issue flow from projects that should make the draft to issue process smoother. 

As for the sunset notice, we published the following banner on May 13th, 2024 on all Classic projects indicating the impending sunset.

![image](https://github.com/user-attachments/assets/0d63f4d2-e9d6-4e90-adcb-0feede78b95d)

**Reply by SandbergSan** [reply]
Good to hear that this might be looked at. This would definately make the new Kanban boards more useable for me.

As for that notice. I dont know why i never saw that notice. Either i am blind or it didnt appear on the kanban board where i usually spent my time. It was not required to go to any other page than the kanban board, when using the Classic Project.

**Reply by xihajun** [reply]
> This is excellent feedback, thank you! I definitely hear you that it would be nice to convert drafts to issues with one click. We have some improvements coming soon for the creating issue flow from projects that should make the draft to issue process smoother.
> 
> As for the sunset notice, we published the following banner on May 13th, 2024 on all Classic projects indicating the impending sunset.
> 
> ![image](https://private-user-images.githubusercontent.com/8809849/365823461-0d63f4d2-e9d6-4e90-adcb-0feede78b95d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzOTE3NTQsIm5iZiI6MTczNDM5MTQ1NCwicGF0aCI6Ii84ODA5ODQ5LzM2NTgyMzQ2MS0wZDYzZjRkMi1lOWQ2LTRlOTAtYWRjYi0wZmVlZGU3OGI5NWQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTZUMjMyNDE0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9Y2NjZDdlYTk3YmRiMjc0OTU2ZTBiM2QzNzU3N2U1YzMyZTJkNGJiOTUzNzI3OGMyZjQ0NTczMDI1OGE5NDdhYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ._7Yoiy8ey4euhCRtyILTe_6n7h8h9zRZhplGOuWn0Ms)

it didn't work with the cards containing images, any idea we can still get it back?
